server/modules/analysis/repository/processingStepCalculationRepository.js

```diff
@@ -89,6 +92,27 @@ export const fetchCalculationAttributeUuidsByChainUuid = async (surveyId, chainU
     R.prop('result'),
   )
 
+/**
+ * Returns an array of calculation attribute definition uuids used by chains different from the specified one
+ */
+export const fetchCalculationAttributeUuidsOtherChainsByChainUuid = async (surveyId, chainUuid, client = db) =>
```

Comment:
can this method be ```fetchCalculationAttributeUuids = async (surveyId, chainUuidExcluded, client = db) ```

Reply:
ok, sure

Reply:
maybe fetchCalculationAttributeUuidsByChainUuidExcluded ?

Reply:
ok